pkg/proxy/proxy.go

```diff
@@ -111,6 +111,7 @@ func (p *proxy) msiHandler(w http.ResponseWriter, r *http.Request) {
 	// This is to keep consistent with the current behavior in pod identity v1 where we
 	// default the client id to the one in AzureIdentity.
 	if clientID == "" {
+		p.logger.Info("client_id not found in request, defaulting to AZURE_CLIENT_ID", "method", r.Method, "uri", r.RequestURI)
```

Comment:
This line looks like it's in the wrong commit?

Reply:
should be fixed now!